dev/sh/build-docs.sh

```diff
@@ -3,7 +3,7 @@
 # Script to build doc site.
 # This script expects to be invoked from the gax-php root.
 #
-# This script will look for the TRAVIS_TAG environment variable, and
+# This script will look for the GITHUB_REF environment variable, and
```

Comment:
nit: may be better to move this to an argument and let the caller specify the ref.

i.e. 
```
<html><head><script>window.location.replace('/gax-php/$1/' + location.hash.substring(1))</script></head><body></body></html>
```

not sure if it needs to be `${1}` in the string...

And in the [action](https://github.com/googleapis/gax-php/pull/301/files#diff-9cf2000c53760d837a449f874e53f792819108d3a4bf346336d0f7d082deae2cR26), `            entrypoint: ./dev/sh/build-docs.sh $GITHUB_REF`

Reply:
done!